vnext/ReactUWP/TurboModule/TurboModuleUtils.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include <ReactUWPABI.h>
+#include <TurboModuleUtils.h>
+#include "../ABI/Instance_rt.h"
+
+namespace react::uwp {
+
+std::shared_ptr<facebook::react::Instance> __cdecl GetReactInstanceFromUwpInstance(
+    const winrt::react::uwp::Instance &uwpInstance) {
+  winrt::com_ptr<ABI::react::uwp::IInstance> abiIInstance;
+  winrt::copy_to_abi(uwpInstance, *abiIInstance.put_void());
+  assert(abiIInstance);
+  ABI::react::uwp::Instance *abiInstance = static_cast<ABI::react::uwp::Instance *>(abiIInstance.get());
```

Comment:
From https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/interop-winrt-abi#unsafe-interop-with-abi-com-interface-pointers, below code may work
```
abiInstance = reinterpret_cast<ABI::react::uwp::Instance*>(get_abi(uwpInstance));
```
Also please do hr result check and null check if needed:
example to check hr result:
```
winrt::check_result(ptr.query_to(winrt::guid_of<To>(), winrt::put_abi(result)));
```

Example to check null:
if (auto reactInstance = abiInstance->getInstance()) {
    return reactInstance->GetInnerInstance();
}